recipes/miceforest/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set version = "2.0.6" %}
+
+package:
+  name: miceforest
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/m/miceforest/miceforest-{{ version }}.tar.gz
+  sha256: 1afe1f5cb598ee4dab5f61817907e4fa692184a84d02cb0edd46b5af6d9eb224
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - scikit-learn
+    - numpy
+    - pandas
+    - seaborn >=0.11.0
+    - matplotlib >=3.3.0
```

Unless you need a pyqt GUI for this package, better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base >=3.3.0
```

---

matplotlib-base doesn't appear to be a package on pypi - will everything run smoothly if there are 2 different dependencies, one for pip and one for conda? No GUIs are used.

---

matplotlib-base is a slightly slimed down version that doesn't depend on qt.